### PR TITLE
feat: 添加了项目基本的类，mapper和数据库字段

### PR DESCRIPTION
**Pull request by ProsperousYe**

### CODE REVIEW

File: src/main/java/com/se2023/backend/entity/User/User.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+    @TableId(value = "id", type = IdType.AUTO)
+    private Integer id;
+    private String username;
```
Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/User/User.java:18]: role and telephone number should also be included

File: src/main/java/com/se2023/backend/entity/Order/Order.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    @TableId(value = "id", type = IdType.AUTO)
+    Integer id;
+
+    String name;
+
+    Integer activityId;
+
+    Integer userId;
+
+    String status;
```
Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/Order/Order.java:24]: status might be set as integer?

Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/Order/Order.java:24]: status might be set as integer?

File: src/main/java/com/se2023/backend/entity/Activity/Activity.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    Integer facilityId;
+
+    Float price;
+
+    String note;
+
+    Integer userAmount;
+
+    String status;
```
Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/Activity/Activity.java:26]: status: string-->integer

File: src/main/java/com/se2023/backend/entity/Activity/Facility.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    String name;
+
+    Integer sportsCentreId;
+
+    Integer capacity;
+
+    String status;
+
+    Integer type;
```
Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/Activity/Facility.java:26]: status: string-->integer/boolean. What is type?

File: src/main/java/com/se2023/backend/entity/Activity/SportsCentre.java [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
 import lombok.NoArgsConstructor;
-import com.baomidou.mybatisplus.annotation.TableId;
-import com.baomidou.mybatisplus.annotation.IdType;
+
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class User {
-    @TableId(type = IdType.AUTO)
```
Comment (sc20jx) [inline src/main/java/com/se2023/backend/entity/Activity/SportsCentre.java:13]: why sports center has user? And there might miss facilities info of sports center?

**Review by sc20jx — APPROVED** [review]

File: src/main/java/com/se2023/backend/entity/User/User.java (hunk 1 above)
Comment (ProsperousYe) [reply src/main/java/com/se2023/backend/entity/User/User.java:18]: 行

File: src/main/java/com/se2023/backend/entity/Order/Order.java (hunk 2 above)
Comment (ProsperousYe) [reply src/main/java/com/se2023/backend/entity/Order/Order.java:24]: 字符串和整形都行

File: src/main/java/com/se2023/backend/entity/Activity/Activity.java (hunk 3 above)
Comment (ProsperousYe) [reply src/main/java/com/se2023/backend/entity/Activity/Activity.java:26]: 布尔值不行

File: src/main/java/com/se2023/backend/entity/Activity/Facility.java (hunk 4 above)
Comment (ProsperousYe) [reply src/main/java/com/se2023/backend/entity/Activity/Facility.java:26]: 区分facility类型，昨天前端讨论之后要加的

File: src/main/java/com/se2023/backend/entity/Activity/SportsCentre.java (hunk 5 above)
Comment (ProsperousYe) [reply src/main/java/com/se2023/backend/entity/Activity/SportsCentre.java:13]: 这个是删掉的代码。。

**Pull request closed without merging**